src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -281,3 +288,31 @@ public partial struct JsonWriterState
         public void WriteStringValue(string value, bool escape = true) { }
     }
 }
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonSerializer
```

Comment:
Branch coverage is 80%-100% depending on file, but somewhat hampered because some tests are disabled because some of the APIs are not yet public (they were made internal). I do need more negative tests.